### PR TITLE
Added vertical_tabs_config, content_lock, formtips, smart_trim module.

### PR DESCRIPTION
**Pull request by saurabh-2k17**

### CODE REVIEW

**Comment by arshadkhan53** [comment]
Hi @saurabh-2k17 can we merge this into single commit (squash)?

File: config/install/content_lock.settings.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    mode: 0
+    values: {  }
+  user:
+    mode: 0
+    values: {  }
+  paragraph:
+    mode: 0
+    values: {  }
+_core:
+  default_config_hash: 6Ka3T1c-CIjEUO1Q64NXKksLthkMhj53a6zy4RJBtkc
```
Comment (vaibhavjain-in) [inline config/install/content_lock.settings.yml]: Remove these 2 lines.

File: config/install/system.action.block_content_break_lock_action.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+uuid: 8e817ed2-2304-4caf-829f-a141f9c034dc
```
Comment (vaibhavjain-in) [inline config/install/system.action.block_content_break_lock_action.yml]: Remove this line.

File: config/install/system.action.node_break_lock_action.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+uuid: 2082bdea-027c-41da-840d-a8b767d372c3
```
Comment (vaibhavjain-in) [inline config/install/system.action.node_break_lock_action.yml]: Remove this line.

File: config/install/user.role.editor.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
+uuid: c23ccd9c-e9f5-4f75-bea1-029eff8e81be
```
Comment (vaibhavjain-in) [inline config/install/user.role.editor.yml]: Remove this file. This file already exists here - https://github.com/srijanone/ezcontent/blob/8.x/modules/ezcontent_user/config/install/user.role.editor.yml
Is there a change that needs to be pushed ?

File: ezcontent.info.yml [hunk 5]
```diff
@@ -13,6 +13,8 @@ install:
   - comment
   - config
   - contact
+  - content_lock
```
Comment (vaibhavjain-in) [inline ezcontent.info.yml]: Please remove this line as its submodule will enable parent module `content_lock`

**Review by vaibhavjain-in — CHANGES_REQUESTED** [review]
@saurabh-2k17 please do changes.

**Review by vaibhavjain-in — APPROVED** [review]
LGTM

**Pull request merged**